src/Microsoft.AspNet.JsonPatch/Adapters/ObjectAdapter.cs

```diff
@@ -542,9 +551,9 @@ public void Copy(Operation<TModel> operation, TModel objectToApplyTo)
 
         private bool CheckIfPropertyExists(
             JsonPatchProperty patchProperty,
-            TModel objectToApplyTo,
-            Operation<TModel> operation,
-            string propertyPath)
+            [NotNull] TModel objectToApplyTo,
+            [NotNull] Operation<TModel> operation,
+            [NotNull] string propertyPath)
```

Comment:
We don't generally do `[NotNull]` on private methods. We want all of these things to be caught as close to the calling code as possible.